docs/TROUBLESHOOTING.md

```
@@ -124,6 +124,62 @@ Failed to download package
 
 ---
 
+### Package Shows as Verification Warning
+
+**Symptoms:**
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)